src/compat.h

```diff
@@ -11,7 +11,10 @@
 #ifndef NOMINMAX
 #define NOMINMAX
 #endif
+#ifdef FD_SETSIZE
```

Comment:
Not signed/unsigned related I just did this based on the windows build log from gitian.